src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -50,8 +50,9 @@ static NodeFunctionInvoker()
             Initialize();
         }
 
-        internal NodeFunctionInvoker(ScriptHost host, BindingMetadata trigger, FunctionMetadata functionMetadata, Collection<FunctionBinding> inputBindings, Collection<FunctionBinding> outputBindings)
```

Comment:
There are additional traces in the Node invoker and other invokers that should also only be traced on primary, e.g. all the "Script for function '{0}' changed. Reloading." type messages, etc.